### PR TITLE
Added examples of usage with PML

### PR DESCRIPTION
**Pull request by AlekseySh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -379,6 +381,109 @@ trainer.fit(pl_model)  # we don't pass loaders to .fit() in DDP
...
+# PML specific
+# criterion = losses.TripletMarginLoss(margin=0.2, triplets_per_anchor="all")
+criterion = losses.ArcFaceLoss(num_classes=df_train["label"].nunique(), embedding_size=model.feat_dim)  # for classification-like losses
+
+sampler = BalanceSampler(train_dataset.get_labels(), n_labels=2, n_instances=2)
+train_loader = torch.utils.data.DataLoader(train_dataset, batch_sampler=sampler)
+
+for batch in tqdm(train_loader):
+    embeddings = model(batch["input_tensors"])
+    loss = criterion(embeddings, batch["labels"])  # PML specific
```
Comment (DaloroAT) [inline README.md:423]: It is necessary for us too

File: docs/readme/usage_with_pml.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+# PML specific
+# criterion = losses.TripletMarginLoss(margin=0.2, triplets_per_anchor="all")
+criterion = losses.ArcFaceLoss(num_classes=df_train["label"].nunique(), embedding_size=model.feat_dim)  # for classification-like losses
+
+sampler = BalanceSampler(train_dataset.get_labels(), n_labels=2, n_instances=2)
+train_loader = torch.utils.data.DataLoader(train_dataset, batch_sampler=sampler)
+
+for batch in tqdm(train_loader):
+    embeddings = model(batch["input_tensors"])
+    loss = criterion(embeddings, batch["labels"])  # PML specific
```
Comment (DaloroAT) [inline docs/readme/usage_with_pml.md:38]: same

**Review by DaloroAT — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -23,6 +23,8 @@ build_readme:
 	cat docs/readme/get_started_config.md >> ${README_FILE}
 	echo "\n## Get started using Python\n" >> ${README_FILE}
 	cat docs/readme/python_examples.md >> ${README_FILE}
+	echo "\n## Usage with PyTorch Metric Learning\n" >> ${README_FILE}
```
Comment (b0nce) [inline Makefile:26]: Why don't we just use `cat *.md > ${README_FILE}`? We can add headers in the individual files and name them in way way the should be arranged: `0_header.md`, `1_faq.md` and so on.

File: README.md (hunk 1 above)
Comment (AlekseySh) [reply README.md:423]: done

**Pull request merged**